subgraphs/hemi-tunnel-withdrawals-subgraph/subgraph.yaml

```diff
@@ -0,0 +1,49 @@
+specVersion: 1.0.0
+indexerHints:
+  prune: auto
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
+    name: L2Bridge
+    network: hemi-sepolia
+    source:
+      abi: L2Bridge
+      address: '0x4200000000000000000000000000000000000010'
+      startBlock: 0
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.7
+      language: wasm/assemblyscript
+      entities:
+        - EvmWithdrawal
```

Comment:
Is this entity needed here? The `entities` prop looks different than L40.

Reply:
yes, I guess with all the changes I forgot of these

Reply:
I checked the [docs](https://github.com/graphprotocol/graph-node/blob/master/docs/subgraph-manifest.md#152-mapping) and it is not entirely clear to me how this is used.

Either way, I removed it and it seems to work. [f81cac3](https://github.com/hemilabs/ui-monorepo/pull/922/commits/f81cac328dd99125a0dc0ece2d69051af5734bdf)